### ISSUE TITLE
xelatex error 32512 (driver return code)

### ISSUE THREAD

**Issue by matthewkidd** [thehogfather/brackets-latex#19]
Hi,

I'm getting a error when trying to compile a tex file with xelatex. Console log below:
![screenshot 2015-02-08 15 31 49](https://cloud.githubusercontent.com/assets/10909064/6096888/a69c1e7a-afa7-11e4-9364-d79655198009.png)

Hope you can help.

Matt

.tex code:
```
\documentclass[12pt]{article}
\usepackage{fontspec}
 
\setmainfont{Times New Roman}
 
 \title{Sample font document}
 \author{Sample Name}
 \date{this month, 2015}
 
\begin{document}
 
 \maketitle
 
 This an \textit{example} of document compiled 
 with \textbf{xelatex} compiler. LuaLaTeX should 
 work fine also.
 
\end{document}
```

**Comment by Wikunia** [comment]
It works on my machine. Do you have any `tex` docs which work on your machine?

**Comment by matthewkidd** [comment]
Yes. The tex doc above works fine when I compile using TeXShop. But I get the screenshot-ed error when I try in Brackets. Problem persists despite a clean install of MacTex. Could I be doing something wrong?

**Comment by Wikunia** [comment]
No I mean do you ever compiled a tex doc using Brackets? Maybe your settings are wrong.

**Comment by matthewkidd** [comment]
Ah sorry. I know pdflatex works. I will go download some sample docs for the other compliers and see how they fare.

**Comment by Wikunia** [comment]
No problem :) I have no other idea at the moment :/

**Comment by matthewkidd** [comment]
Thanks for the help and quick responses anyway. What were the settings you thought might be wrong? Also, the other compliers work fine – only xelatex is being a troublemaker.

**Comment by Wikunia** [comment]
You only have one `Tex path` in your settings were all compilers should exist. But I don't know the error `driver return code...` so I don't know. Maybe @thehogfather has an idea cause it's his extension :D

**Comment by matthewkidd** [comment]
Oh wow. I'm such an idiot. Ok, so I've fixed the issue! I was changing the compiler to ```xelatex```  here:
![screenshot 2015-02-08 18 58 56](https://cloud.githubusercontent.com/assets/10909064/6097606/9284b0f6-afc4-11e4-9da1-c3d7a9eb6fed.png)

But I neglected to also changed the settings in LaTeX Settings:
![screenshot 2015-02-08 18 59 29](https://cloud.githubusercontent.com/assets/10909064/6097610/a77e0b74-afc4-11e4-9841-560c795966ab.png)

Thanks for your help Wikunia! Case closed.
